share/spice/flights/route/flights_route.css

```diff
@@ -1,5 +1,23 @@
 .tile--route  {
-    width: 37em;
+    padding: 1.44em;
+}
+
+.zci--route .inactive  {
+    background-color: #f7f7f7;
```

Comment:
We'll need to use our `bg--*` classes here so this looks good in the dark theme as well. The classes swap the colours for the dark themes automatically.

@brianrisk how does it currently look in the dark theme?

/cc @thm we don't have a color class for `#e4e4e4` though, but we have `bg--platinum-light` which is `#e5e5e5`

![duckduckgo_style_guide](https://cloud.githubusercontent.com/assets/873785/11353439/1908430c-9211-11e5-9a0b-4840c4878972.png)